### PR TITLE
ETL for Violence reporting form

### PR DESCRIPTION
**Pull request by gabriel090**
ETL for Violence reporting form

### CODE REVIEW

File: omod/src/main/resources/sql/Scheduled_Updates.sql [hunk 1]
```diff
@@ -6582,6 +6582,179 @@ order by e.patient_id
...
+order by e.patient_id
+        ON DUPLICATE KEY UPDATE visit_date=VALUES(visit_date),
+          provider=VALUES(provider),
+          place_of_incident=VALUES(place_of_incident),
+          date_of_incident=VALUES(date_of_incident),
+          time_of_incident=VALUES(time_of_incident),
+          abuse_against=VALUES(abuse_against),
+          form_of_incident=VALUES(form_of_incident),
+          perpetrator=VALUES(perpetrator),
+          other_perpetrator=VALUES(other_perpetrator),
```
Comment (njorocs) [inline omod/src/main/resources/sql/Scheduled_Updates.sql:6724]: other_perpetrator does not exist. Your test for etl refresh should have failed.

File: omod/src/main/resources/sql/dwapi/DML.sql [hunk 2]
```diff
@@ -5907,6 +5907,134 @@ group by o.id order by o.concept_id;
...
+visit_date,
+location_id,
+encounter_id,
+place_of_incident,
+date_of_incident,
+time_of_incident,
+abuse_against,
+form_of_incident,
+perpetrator,
+--other_perpetrator,
```
Comment (njorocs) [inline omod/src/main/resources/sql/dwapi/DML.sql:5930]: remove this line

**Review by njorocs — CHANGES_REQUESTED** [review]

**Review by njorocs — APPROVED** [review]
lgtm

File: omod/src/main/resources/sql/DML.sql [hunk 3]
```diff
@@ -6156,6 +6156,132 @@ group by o.id order by o.concept_id;
...
+visit_date,
+location_id,
+encounter_id,
+place_of_incident,
+date_of_incident,
+time_of_incident,
+abuse_against,
+form_of_incident,
+perpetrator,
+--other_perpetrator,
```
[the remark this answers is not in the document]
Comment (gabriel090) [reply omod/src/main/resources/sql/DML.sql]: fixed

File: omod/src/main/resources/sql/Scheduled_Updates.sql (hunk 1 above)
Comment (gabriel090) [reply omod/src/main/resources/sql/Scheduled_Updates.sql]: Resolved

File: omod/src/main/resources/sql/dwapi/DML.sql (hunk 2 above)
Comment (gabriel090) [reply omod/src/main/resources/sql/dwapi/DML.sql]: Fixed

**Pull request merged**